packages/hoppscotch-sh-admin/src/pages/users/invited.vue

```diff
@@ -100,4 +69,26 @@ const getCreatedTime = (date: string) => format(new Date(date), 'hh:mm a');
 // Get Invited Users
 const { fetching, error, data } = useQuery({ query: InvitedUsersDocument });
 const invitedUsers = computed(() => data?.value?.admin.invitedUsers);
+
+// The new invited users list that is used in the table
+const newInvitedUsersList = computed(() => {
```

Comment:
We could perhaps rename it to better suit the purpose. Something along the lines of `invitedUsersListToRender`.

Reply:
On second thought, do we need this computed property? Can't we supply `invitedUsersList` straightaway?